index.md

```diff
@@ -248,33 +248,33 @@ and try out PayEx Checkout for yourself!' %}
 
 ## Here is an unordered list
 
-*   Item foo
-*   Item bar
-*   Item baz
-*   Item zip
+* Item foo
+* Item bar
+* Item baz
+* Item zip
 
 ## And an ordered list
 
-1.  Item one
-2.  Item two
-3.  Item three
-4.  Item four
+1. Item one
+2. Item two
+3. Item three
+4. Item four
 
 ## And a nested list
 
-*   level 1 item  
-    *   level 2 item
-    *   level 2 item
-    *   level 3 item
-    *   level 3 item
-*   level 1 item  
-    *   level 2 item
-    *   level 2 item
-    *   level 2 item
-*   level 1 item  
-    *   level 2 item
-    *   level 2 item
-*   level 1 item
+* level 1 item  
+  * level 2 item
+  * level 2 item
+  * level 3 item
+  * level 3 item
+* level 1 item  
+  * level 2 item
+  * level 2 item
+  * level 2 item
+* level 1 item  
+  * level 2 item
+  * level 2 item
+* level 1 item

```

Comment:
Please avoid modifying content that is not a part of the PR. I suppose this happens because you're not using the RemarkJS plugins to format Markdown code?